ember-toucan-styles/package.json

```diff
@@ -62,7 +62,7 @@
     "tailwindcss": "^2.2.15 || ^3.0.0"
   },
   "devDependencies": {
-    "@babel/core": "7.18.2",
+    "@babel/core": "7.20.12",
```

Comment:
Both this and `eslint-plugin-ember` needed updating due to our ESLint configs getting updated and having unmet peer dependencies. `pnpm` FTW!

Reply:
Does `pnpm` tell you what needs updating?

Reply:
@nicolechung yeah!  When running `pnpm i` you'll get "unmet peer dependency" lines like:

```bash
│ ├─┬ some-package
│ │ ├── ✕ missing peer some-missing-peer@^v.v.v
```